src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -280,6 +318,16 @@ public AzureAppConfigurationOptions ConfigureClientOptions(Action<ConfigurationC
             return this;
         }
 
+        /// <summary>
+        /// Configure the time to wait for a response from one replica before initiating a request to the next available replica.
+        /// </summary>
+        /// <param name="parallelRetryInterval">The time to wait for a response from one replica before trying the next available replica.</param>
+        public AzureAppConfigurationOptions ConfigureParallelRetryTimeout(TimeSpan parallelRetryInterval)
+        {
+            this.ParallelRetryInterval = parallelRetryInterval;
```

Comment:
> this.ParallelRetryInterval = parallelRetryInterval; [](http://example.com/codeflow?start=11&length=52)

We should also add a min ParallelRetryInterval of 1 second (like our min refresh interval).

Reply:
I am assuming we need to add a validation that the retry interval should be greater than 1 second.
correct?

Reply:
Exactly.